packages/legend-application-studio/src/components/editor/editor-group/GrammarTextEditor.tsx

```diff
@@ -974,6 +987,200 @@ export const GrammarTextEditor = observer(() => {
       );
   }
 
+  //clear all inline folding decorations
```

Comment:
all of these helper functions, if possible, should be declared outside of the render method